dotnet-efcore/EntityFrameworkCoreVsDapper/EntityFrameworkCoreVsDapper/Repositories/PersonsRepositoryDapper.cs

```diff
@@ -0,0 +1,52 @@
+﻿using BenchmarkDotNet.Attributes;
+using Dapper;
+using EntityFrameworkCoreVsDapper.Models;
+using System.Data.SqlClient;
+
+namespace EntityFrameworkCoreVsDapper.Repositories
+{
+    public class PersonsRepositoryDapper : IPersonsRepository
+    {
+        //ADD THE CONNECTION STRING TO YOUR DATABASE HERE
+        private string ConnectionString = "";
+
+        [Benchmark]
+        public List<Person> Get_1000_Persons()
+        {
+            using (var dbConnection = new SqlConnection(ConnectionString))
+            {
+                dbConnection.Open();
+                List<Person> items = (List<Person>)dbConnection.Query<Person>("SELECT TOP(1000) * FROM Persons");
+                dbConnection.Close();
+
+                return items;
+            }
+        }
+
+        [Benchmark]
+        public List<Person> Get_10000_Persons()
+        {
+            using (var dbConnection = new SqlConnection(ConnectionString))
```

Comment:
You can switch to the using declaration to reduce nesting since the project is targetting .NET 7